### ISSUE TITLE
RFE: Show server IP/FQDN 

### ISSUE THREAD

**Issue by MagicFab** [bitfireAT/davdroid#420]
Under Android Settings > Accounts I only see the name I gave to the accounts when initially configured.

As I have several CalDav and DavDroid accounts I configured (almost a dozen) for work, several customers, personal use, family, etc. it becomes difficult to differentiate them.

**Comment by rfc2822** [comment]
Android Settings > Accounts always shows only the account name.

**Comment by MagicFab** [comment]
Precisely, it would be most useful to also show the ip address and FQDN for each account.

**Comment by MagicFab** [comment]
In Android accounts the FQDN and / or IP could be added to the description. The information could be in the DAVDroid app anyways.

This issue is not about editing DAVDroid accounts, only about displaying detailed information about them. Implementing this would be less disruptive than editing (#154) and less risky, and tracking its progress separately can help others find it and contribute to it when/if possible. Please consider leaving this open for tracking / other contributors.

**Comment by rfc2822** [comment]
To show info about DAVdroid accounts, there has to be a GUI for managing DAVdroid accounts first. This is why this is related to #154.
